exporter/loadbalancingexporter/README.md

```diff
@@ -117,11 +121,80 @@ exporters:
         - backend-2:4317
         - backend-3:4317
         - backend-4:4317
-      # Notice to config a headless service DNS in Kubernetes  
+      # Notice to config a headless service DNS in Kubernetes
       # dns:
-      #  hostname: otelcol-headless.observability.svc.cluster.local        
+      #  hostname: otelcol-headless.observability.svc.cluster.local
+
+service:
+  pipelines:
+    traces:
+      receivers:
+        - otlp
+      processors: []
+      exporters:
+        - loadbalancing
+    logs:
+      receivers:
+        - otlp
+      processors: []
+      exporters:
+        - loadbalancing
+```
+
+Persistent queue, retry and timeout usage example:
+
+```yaml
+receivers:
+  otlp:
+    protocols:
+      grpc:
+        endpoint: localhost:4317
+
+processors:
+
+exporters:
+  loadbalancing:
+    timeout: 10s
+    retry_on_failure:
+      enabled: true
+      initial_interval: 5s
+      max_interval: 30s
+      max_elapsed_time: 300s
+    sending_queue:
+      # please take a note that otlp.sending_queue will be
+      # disabled automatically in this case to avoid data loss
+      enabled: true
+      num_consumers: 2
+      queue_size: 1000
+      storage: file_storage/otc
+    routing_key: "service"
+    protocol:
+      otlp:
+        # all options from the OTLP exporter are supported
+        # except the endpoint
+        timeout: 1s
+        # doesn't take any effect because loadbalancing.sending_queue
+        # is enabled
```

Comment:
This is still confusing to me: looking at the code in the factory, the only property it overrides is the endpoint. Why is this not effective?

```go
func buildExporterConfig(cfg *Config, endpoint string) otlpexporter.Config {
	oCfg := cfg.Protocol.OTLP
	oCfg.Endpoint = endpoint

	return oCfg
}
```

Reply:
That was a leftover from previous version, removed